pal/ios-osx/tlsio_appleios.c

```diff
@@ -725,14 +726,39 @@ static int tlsio_appleios_setoption(CONCRETE_IO_HANDLE tls_io, const char* optio
         /* Codes_SRS_TLSIO_30_122: [ If the value parameter is NULL, tlsio_appleios_setoption shall do nothing except log an error and return FAILURE. ]*/
         /* Codes_SRS_TLSIO_OPENSSL_COMPACT_30_520 [ The tlsio_setoption shall do nothing and return FAILURE. ]*/
         TLSIO_OPTIONS_RESULT options_result = tlsio_options_set(&tls_io_instance->options, optionName, value);
-        if (options_result != TLSIO_OPTIONS_RESULT_SUCCESS)
+        if (options_result == TLSIO_OPTIONS_RESULT_SUCCESS)
         {
-            LogError("Failed tlsio_options_set");
-            result = __FAILURE__;
+            result = 0;
+        }
+        else if (options_result == TLSIO_OPTIONS_RESULT_NOT_HANDLED)
+        {
+            /* Pass through */
+            if (strcmp(optionName, "tcp_nodelay") == 0)
+            {
+                CFDataRef inputRawData = CFReadStreamCopyProperty(tls_io_instance->sockRead, kCFStreamPropertySocketNativeHandle);
+                CFSocketNativeHandle inputRawHandle;
```

Comment:
Does this handle not also need to be released? According to the documentation for CFDataGetBytes, it returns a copy

Reply:
thx for noticing this, but the `CFSocketNativeHandle` is actually a `int`, so it doesn't need to release.

[doc](https://developer.apple.com/documentation/corefoundation/cfsocketnativehandle?language=objc) says: `typedef int CFSocketNativeHandle;`